resources/views/components/menu/footer.blade.php

```diff
@@ -21,13 +21,11 @@
                 'url' => 'https://docs.retroachievements.org/FAQ',
                 'visible' => true,
             ],
-            // [
-            //     'active' => Route::is('docs.api'),
-            //     'label' => __('API'),
-            //     'url' => url('APIDemo.php'),
-            //     // 'url' => route('docs.api'),
-            //     'visible' => true,
-            // ],
+            [
+                'label' => __('API'),
+                'url' => url('docs/api'),
```

Comment:
This can be `'url' => route('docs.api'),` now because the named route is actually usable.
The `active` check could stay, too - not sure if it actually does something atm 🤔

Reply:
Makes sense. I've made these adjustments. The `active` check doesn't seem to be doing anything presently unless I'm mistaken.